panel/src/components/Navigation/Topbar.vue

```diff
@@ -146,7 +146,8 @@ export default {
       }
     },
     unregistered() {
-      return !this.$store.state.system.info.license ? true : false;
+      let license = this.$store.state.system.info.license;
```

Comment:
I wouldn't do this in the panel, but in the System class instead.

Reply:
Do you mean the `$system->license()` method should return `false` if there is a license but not a valid one? That would mean that the Panel would display the "unregistered demo" warning even though there is a license, just not a valid one. It's also not really semantic in my opinion.

I think we should keep this in the Panel code, but in a central location where both the topbar and the settings page can access it. In the case "license is there, but not valid for the current Kirby installation", the following would happen:

- The topbar displays the buttons as the installation is unregistered.
- The settings page displays the license code, but with a warning next to it that it is not valid.

What do you think?

PS: Maybe we could do this after the launch though, as it will only become relevant once we generate a different type of license. So if you think that the idea itself is the right one, I will remove this change from the PR and create an issue in the ideas repo.